contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -485,6 +485,36 @@ open class ContourLayout(
         lambda = unwrapXIntLambda(provider)
     )
 
+  fun matchX(
+    view: View,
+    marginLeft: Int = 0,
+    marginRight: Int = 0
+  ): XAxisSolver {
+    return leftTo { view.left() + marginLeft }.rightTo { view.right() - marginRight }
+  }
```

Comment:
nit: maybe `alignTo` cuz not parent view?

Reply:
I think `match` still makes sense as in matching the position to another view, especially since it has very similar functionality to `matchParent`. `alignTo` seems a bit like positioning the view relative to another view to me

Reply:
I'd like to rename `matchN` to `matchNWith` or `matchNTo` ?

Reply:
I personally like `matchXOf(otherView)`?

Reply:
`matchXTo` seems to have to correct preposition in the context of matching this view's position _to_ another view